erp/migrations/0116_auto_20210419_1309.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 3.2 on 2021-04-19 11:09
+
+from django.db import migrations, models
+from erp.models import Accessibilite as Access
+from erp import schema
+
+
+def separate_fields(apps, schema_editor):
+    Accessibilite: Access = apps.get_model("erp", "Accessibilite")
+    for access in Accessibilite.objects.all():
+        if access.cheminement_ext_pente is None:
+            pass
+        elif access.cheminement_ext_pente == schema.PENTE_AUCUNE:
+            access.cheminement_ext_pente = False
```

Comment:
I suspect a field cannot accept either a boolean or a string at a given point in times :)